CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Changed
 
+- Made Flutter project open to project view by default (#8573)
```

Comment:
Add trailing period?

Reply:
Argh sorry, I forgot my own suggested rules and put an autosubmit on this. There's a bunch of inconsistencies in this file, so I will make a follow up PR.